src/main/java/frc/robot/swerve/SwerveModule.java

```diff
@@ -19,6 +19,7 @@
 import frc.robot.misc.util.sensors.SensorUnitConverter;
 
 public class SwerveModule {
+  private static final double MAX_VELOCITY = 4.5;
```

Comment:
I believe we have the SDS MK4 L1 modules, which would run at 4.1148 Meters per second.

https://www.swervedrivespecialties.com/products/mk4-swerve-module

Reply:
this number should also be converted to feet/sec since we're not using SI anymore